crates/tauri-plugin-service/android/src/main/java/com/plugin/holochain_service/HolochainServicePlugin.kt

```diff
@@ -104,6 +105,24 @@ class HolochainServicePlugin(
         }
     }
 
+    /*
+     * Set the runtime network config
+     *
+     * This will be applied the next time `start` is called.
+     */
+    @Command
+    fun setConfig(invoke: Invoke) {
```

Comment:
Add a new tauri plugin command to set the config, which is saved to the plugin state, and then applied when starting the runtime.